presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -267,6 +293,113 @@ private Plan plan(String sql, LogicalPlanner.Stage stage)
         }
     }
 
+    private static void assertEqualPlans(Plan actual, Plan expected)
```

Comment:
I use this one as well. It allows to compare the resulting tree with some pre-defined pattern. In my case I also want  to compare the plan agains another query's plan which should be identical when created and optimized.